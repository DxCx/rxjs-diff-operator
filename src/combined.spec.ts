'use strict';

import 'jest';

import { Observable } from 'rxjs';
import './index';

describe('diff operator combined', () => {
    it.only('works togather', () => {
        let obs: Observable<number> = Observable.of(1, 2, 3, 4, 5);

        return obs.toDiff().fromDiff().bufferCount(6).toPromise().then((msgs) => {
            expect(typeof msgs).toBe('object');
            expect(msgs).toMatchSnapshot();
        });
    });
});
