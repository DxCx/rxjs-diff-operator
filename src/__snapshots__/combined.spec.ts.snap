exports[`diff operator combined works togather 1`] = `
Array [
  1,
  2,
  3,
  4,
  5
]
`;
